content/site-policy/privacy-policies/github-general-privacy-statement.md

```diff
@@ -60,7 +60,7 @@ Personal Data is collected from you directly, automatically from your device, an
 * Essential Cookies and Similar Tracking Technologies: We use cookies and similar technologies to provide essential functionality like storing settings and recognizing you while using our Services.
 * Non-essential Cookies: Depending on your jurisdiction, we may use online analytics products that use cookies to help us analyze how de-identified users use our Services and to enhance your experience when you use the Services. We may also employ third-party Cookies to gather data for interest-based advertising. In some jurisdictions, we only use non-essential cookies after obtaining your consent. See [this](#what-are-your-cookie-choices-and-controls) section for more details and control options.
 * Email Marketing Interactions: Our emails may have web beacons that offer information on your device type, email client, email reception, opens, and link clicks.
-* Geolocation Information: Depending on the Service's functionality, we collect regional geolocation data
+* Geolocation Information: Depending on the Service's functionality, we collect regional geolocation data.
```

Comment:
I know this is frustrating but let's leave this out of this PR: I'll raise it separately with our Site Policy team

Reply:
Ha, understood. It was the one thing that jumped off the page to my eyes and was actually what led me to start the PR :sweat_smile: . I will revert this commit momentarily.

Reply:
I should have asked @hdub-tech but if you'd prefer to raise it yourself, you can do so here https://github.com/github/site-policy/blob/main/CONTRIBUTING.md

Reply:
Oh is there not already an internal issue for it? Either way, I can open a public one for you if you did not already beat me to it.

Reply:
I haven't opened one yet; I wanted to give you the option to open one publicly, so you'd get the full credit for it. Either way just let me know here 👍

Reply:
Apologies on the delay, I didn't receive a notification for your comment! Here is the issue: https://github.com/github/docs/issues/36556